app.js

```diff
@@ -0,0 +1,39 @@
+const express = require('express');
+const morgan = require ('morgan');
+const userRoutes = require('./routes/userRoutes')
+const eventsRoutes = require('./routes/eventRoutes');
+const purchasRouter = require('./routes/purchasRoutes')
```

Comment:
יש לך כאן טעות כתיב. זה גם ידרוש ממך לשנות את שם הקובץ
```suggestion
const purchaseRouter = require('./routes/purchaseRoutes')
```